js/Course.js

```diff
@@ -15,12 +15,12 @@ function Course(quarter, name, nameID, gt, units,
                 days, hour1, min1, hour2, min2, bldg, room) {
 
     //General information
-    this.quarter = quarter; //written as a string in the format of "season####"
+    this.quarter = (quarter === "" ? "None provided" : quarter); //written as a string in the format of "season####"
```

Comment:
It doesn't check for absolute equality. == checks for equality between any two types and === checks for equality for specifically the same type. 
![image](https://cloud.githubusercontent.com/assets/7998948/10925921/3c6a5a94-8247-11e5-84fd-a35833a332ed.png)